README.rst

```diff
@@ -59,16 +59,11 @@ To update, run::
 
     pip install -U crash
 
-If you are not using Python version 3.5 or above, recent version of `pip`_ will
-install an earlier version of Crash. This is because newer versions of this
-package are not compatible with Python 2.7 or 3.4 and below.
-
 Standalone
 ----------
 
 Crash is also available as a standalone executable that includes all the
-necessary dependencies, and can be run as long as Python (>= 3.5) is available
-on your system.
+necessary dependencies.
```

Comment:
Shouldn't we keep mentioning the minimal required python version for the bundles?

Reply:
Oh, maybe I was too quick to remove this paragraph. I've just brought it back. Thanks for spotting.